### PR TITLE
iulia.buta

### PR DESCRIPTION
**Pull request by ButaIulia**
### Pull Request Overview

This pull request adds the Project Documentation for Fire Alarm.

### TODO or Help Wanted

This pull request is still a draft.Further modifications of the documentation will be uploaded here.

### Author

Signed-off-by: Iulia Buta <iuliamirelabuta@gmail.com>

### CODE REVIEW

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+The format is
+
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+
+
+-->
+
+| Device                                                                                                  | Usage               | Price                                                                                                                                                                                                                                                                                |
+| ------------------------------------------------------------------------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:89]:
```suggestion
|-|-|-|
```

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+| [6 Father-Father wires]()                                                                               | Connectors          | [0 RON(Already had this)]()                           |     
+| [ RGB LED Module]()                                                                                          |        | [0 RON(Already had this)]()                                                                                                                                                                                                                               
+
+## Software
+
+| Library                                  | Description                        | Usage                              |
+| ---------------------------------------- | ---------------------------------- | ---------------------------------- |
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:102]:
```suggestion
|-|-|-|
```

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+| [st7789] | Display driver for st7789 | Used for the display for Pico Explorer |
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:103]: Do you need this library?

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+| [Rust](https://www.rust-lang.org/)       | Programming Language               | Programming Language               |
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:104]: Rust is not a library.

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+| [Embassy](https://embassy.dev/)          | Framework                          | Framework                          |
+| [embedded-graphics]         | 2D graphics library                         | Used for drawing to the display
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:106]: Do you need this library?

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
+# Fire alarm system
+
+:::info
+
+*Author*: Buta Iulia-Mirela
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:5]:
```suggestion
*Author*: Buta Iulia-Mirela \
```

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
+# Fire alarm system
+
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:2]: Add a one line description of your project.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+         +-----------------+
+         | Raspberry Pi Pico|
+         +-----------------+
+                 |
+      +----------+----------+
+      |                     |
+      V                     V
+ +---------+           +-----------+
+ |  Buzzer |           |  ST7789   |
+ +---------+           +-----------+
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:76]:
````suggestion
```
+-------------------------+
 |      Fire Sensor        |
 +-------------------------+

                 |
                 | (Signal)
                 V
         +-----------------+
         | Raspberry Pi Pico|
         +-----------------+
                 |
      +----------+----------+
      |                     |
      V                     V
 +---------+           +-----------+
 |  Buzzer |           |  ST7789   |
 +---------+           +-----------+
```
````

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+The format is
+
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+
+
+-->
+
+| Device                                                                                                  | Usage               | Price                                                      
+| [Rapspberry Pi Pico](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [39 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html)  |
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:91]: Please add a link to all the parts.

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+| [Buzzer] |Buzzer | [0 RON (Already had this)]|  
+| [4 Mother-Father wires 30 cm]  | Connectors          | [0 RON(Already had this)]   |
+| [6 Father-Father wires]| Connectors          | [0 RON(Already had this)] |     
+| [ RGB LED Module]   |    LED  | [0 RON(Already had this)]|                                   
+
+## Software
+
+| Library                                  | Description                        | Usage                              |
+|-|-|-|
+| [VsCode](https://code.visualstudio.com/) | Integrated Development Environment | Integrated Development Environment |
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:104]:
```suggestion     
```

VS Code is not a library.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+
+## Connection Overview:
+* The fire sensor is connected to one of the GPIO pins of the Raspberry Pi Pico for detecting fire or smoke.
+* The buzzer is connected to another GPIO pin to sound the alarm when triggered.
+* The ST7789 display is connected to the Raspberry Pi Pico via SPI for displaying system status and alerts.
+* Power supply is connected to Raspberry Pi Pico for powering the entire system.
+
+### Schematics
+
+-+-------------------------+
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:60]: This for sure does not render corectly, please test it, you are missing a ```.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
++ +-------------------------+
++
++                 |
++                 | (Signal)
++                 V
++         +-----------------+
++         | Raspberry Pi Pico|
++         +-----------------+
++                 |
++      +----------+----------+
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:89]: This is not correct and does not render, this is the git diff that the website shows. You can clock click *Commit Changes* and it makes the modifications. Please make the ascii chart enclosed in ```.

File: docs/project/2024/iulia_mirela.buta/index.md (hunk 9 above)
Comment (alexandruradovici) [reply docs/project/2024/iulia_mirela.buta/index.md:110]: Links are still missing and the square bracket are not useful.

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+| Library                                  | Description                        | Usage                              |
+|-|-|-|
+
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. https://how2electronics.com/fire-alarm-system-using-flame-sensor-raspberry-pi-pico/
+2. https://www.youtube.com/watch?v=y1_scz9I0go
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:130]: Please use the proper markdown links syntax.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/iulia_mirela.buta/index.md [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+                 +-----------------+
+                 | Raspberry Pi Pico|
+                 +-----------------+
+                         |
+              +----------+----------+
+              |                     |
+              V                     V
+         +---------+           +-----------+
+         |  Buzzer |           |  ST7789   |
+         +---------+           +-----------+
```
Comment (alexandruradovici) [inline docs/project/2024/iulia_mirela.buta/index.md:76]:
````suggestion
```
+-------------------------+
|      Fire Sensor        |
+-------------------------+

                         |
                         | (Signal)
                         V
                 +-----------------+
                 | Raspberry Pi Pico|
                 +-----------------+
                         |
              +----------+----------+
              |                     |
              V                     V
         +---------+           +-----------+
         |  Buzzer |           |  ST7789   |
         +---------+           +-----------+
```
````

Comment (alexandruradovici) [reply docs/project/2024/iulia_mirela.buta/index.md:76]: Use *Commit suggestion* to merge this and then udate your branch with `git pull`.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]
No architecture diagram?

**Comment by ButaIulia** [comment]
> No architecture diagram?

It's not the one that I put already? With the arrows and how are connected between them?

**Review by DanutAldea — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**